packages/earn-controller/package.json

```diff
@@ -47,10 +47,12 @@
     "since-latest-release": "../../scripts/since-latest-release.sh"
   },
   "dependencies": {
+    "@ethersproject/bignumber": "^5.7.0",
     "@ethersproject/providers": "^5.7.0",
     "@metamask/base-controller": "^8.0.1",
     "@metamask/controller-utils": "^11.9.0",
-    "@metamask/stake-sdk": "^1.0.0"
+    "@metamask/stake-sdk": "3.2.0",
```

Comment:
Oops, noticed one more thing. Is there a reason this is an exact version? It might be more useful to use `^3.2.0`, this way if there is some bugfix you need to make to this package, you don't have to release a new version of `earn-controller` to get that fix into the clients, you can just upgrade this package.

Reply:
This is a mistake thanks!